### ISSUE TITLE
Final Review

### ISSUE THREAD

**Issue by SiyuSiyuYang** [andrewkozma/4741-Project#23]
What I liked about the project:
- As a sports fan, I can relate that this seems to be a natural idea people have watching the game -- can we find anything from all these awesome game data? Even though the final result is not every satisfactory, the team did try many methods and combine some unsupervised techniques that are not covered in class to work on this problem. I also enjoy the flow of logic of the report.

Things that can be improved in the future:
- When choosing the model, the authors seem to worry more about overfitting, which seems to be less of a problem. Given the probable correlation between features, I would choose loss functions that introduce sparsity rather than prevent overfitting.
